pkgs/by-name/ds/dstep/package.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, stdenv, fetchzip }:
+stdenv.mkDerivation rec {
+  name = "dstep";
+  version = "1.0.0";
+
+  src = passthru.sources.${stdenv.hostPlatform.system} or (throw
+    "Unsupported system: ${stdenv.hostPlatform.system}");
+
+  passthru = {
```

Comment:
Why this strange use of `passthru`?

`passsthru` is employed when we do want to pass information to the outside world without triggering a rebuild.

Using it to pass the sources - that are expected as an input - goes completely out of its purposes.